harper-core/src/linting/save_to_safe.rs

```diff
@@ -0,0 +1,77 @@
+use crate::{
+    Token,
+    linting::{Lint, LintKind, PatternLinter, Suggestion},
+    patterns::{InflectionOfBe, OwnedPatternExt, Pattern, SequencePattern, Word},
+};
+
+pub struct SaveToSafe {
+    pattern: Box<dyn Pattern>,
+}
+
+impl Default for SaveToSafe {
+    fn default() -> Self {
+        let pattern = SequencePattern::default()
+            .then(InflectionOfBe::new().or(Box::new(Word::new("it"))))
+            .then_whitespace()
+            .t_aco("save")
+            .then_whitespace()
+            .t_aco("to")
+            .then_whitespace()
+            .then_verb();
```

Comment:
You might want to check if the verb is also an adjective or noun, because those could be false positive. You really only want infinitive forms of verbs too, the only form that can come after "to". Past and continuous forms can act as adjectives and nouns. I can't think of examples off the top of my head right now though.